charts/datree-admission-webhook/values.yaml

```diff
@@ -58,6 +58,7 @@ resources: {}
 #   memory: 48Mi
 nodeSelector: {}
 affinity: {}
+tolerations:
```

Comment:
I see that the initial val suppose to be `tolerations: []` according to prometheus example:
https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus/values.yaml